overlay/libexec/k3os/mode-local

```diff
@@ -10,4 +10,12 @@ setup_ssh()
     ln -s /var/lib/rancher/k3os/ssh /etc/ssh
 }
 
+setup_rancher_node()
+{
+    mkdir -p /etc/rancher
+    mkdir -p /var/lib/rancher/k3os/node
```

Comment:
```suggestion
    mkdir -p /var/lib/rancher/k3os/etc-rancher
```

The information that needs to be persisted is in `/etc/rancher/node/`, the sub-directory would then be `/var/lib/rancher/k3os/node/node` which seems a little weird, so maybe a good idea to make the parent directory name more descriptive of the use.

Reply:
@erikwilson note the trailing slash: `ln -s /var/lib/rancher/k3os/node /etc/rancher/`

on a local instance:
```
k3os-27078 [~]$ grep VERSION_ID /etc/os-release 
VERSION_ID="7fa0889"
k3os-27078 [~]$ cat /run/k3os/mode 
local
k3os-27078 [~]$ realpath /etc/rancher/node/password
/var/lib/rancher/k3os/node/password
k3os-27078 [~]$ 
```

Reply:
This kinda comes down to:
- do we want/need to persist all of `/etc/rancher`?
- or is `/etc/rancher/node` enough?

I'd like a flag in k3s where I can tell it where to persist and find this kind of information, but failing that I am okay with a little targeted symlink magic.